dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -519,48 +519,77 @@ ANNO_CLASSSOURCE_ENTRY_ADAPT_EXCEPTION.useraction=No user action is available.
-
+# # {0} A number: The count of module locations which had Jandex indexes.
+# # {1} A number: The total count of module locations.
+# # {2} A number: The count of module classes which were read from a Jandex index.
+# # {3} A number: The total count of module classes.
+# # {4} A string: A module name
+ANNO_JANDEX_USAGE=CWWKC0092I: Jandex coverage of module {4}: Jandex indexes were read for {0} out of {1} module locations; Jandex indexes provided {2} out of {3} module classes.
+ANNO_JANDEX_USAGE.explanation=This shows the proportion of a module which was scanned for annotations using Jandex indexes.  Module locations are either the module itself, or, for a Web module, are the WEB-INF/classes folder and WEB-INF/lib Jar files of the module.
+ANNO_JANDEX_USAGE.useraction=No action is required if Jandex indexes are intended to be read for the application.  To ignore Jandex indexes for a single application, then set the useJandex="false" attribute in the 'application' element of that application in server.xml. Alternatively, to ignore Jandex indexes in all applications, set useJandex="false" in the 'applicationManger' element in server.xml.
+
+# # {0} A string: An application name.
+# # {1} A string: A module name.
+# # {2} A number: The total count of module locations.
+# # {3} A number: The count of module locations which had Jandex indexes.
+# # {4} A number: The total count of module classes.
+# # {5} A number: The count of module classes which were read from a Jandex index.
+ANNOCACHE_JANDEX_USAGE=CWWKC0093I: Jandex coverage of module {1} in application {0}: Jandex indexes were read for {3} out of {2} module locations; Jandex indexes provided {5} out of {4} module classes.
+ANNOCACHE_JANDEX_USAGE.explanation=This shows the proportion of a module which was scanned for annotations using Jandex indexes.  Module locations are either the module itself, or, for a Web module, are the WEB-INF/classes folder and WEB-INF/lib Jar files of the module.
+ANNOCACHE_JANDEX_USAGE.useraction=No action is required if Jandex indexes are intended to be read for the application.  To ignore Jandex indexes for a single application, then set the useJandex="false" attribute in the 'application' element of that application in server.xml. Alternatively, to ignore Jandex indexes in all applications, set useJandex="false" in the 'applicationManger' element in server.xml.
+
+# # {0} A string: Message text from an exception.
+ANNO_TARGETS_CACHE_EXCEPTION=CWWKC0101W: Annotation processing cache error: {0}
+ANNO_TARGETS_CACHE_EXCEPTION.explanation=An exception occured while accessing annotations cache data.
+ANNO_TARGETS_CACHE_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.